src/VisualStudio/Core/Test.Next/Services/AssetProviderTests.cs

```diff
@@ -136,7 +136,7 @@ public async Task TestProjectSynchronization()
             var assetSource = new SimpleAssetSource(workspace.Services.GetService<ISerializerService>(), map);
 
             var service = new AssetProvider(sessionId, storage, assetSource, remoteWorkspace.Services.GetService<ISerializerService>());
-            await service.SynchronizeProjectAssetsAsync(new HashSet<Checksum> { await project.State.GetChecksumAsync(CancellationToken.None) }, CancellationToken.None);
+            await service.SynchronizeProjectAssetsAsync(await project.State.GetStateChecksumsAsync(CancellationToken.None), CancellationToken.None);
```

Comment:
made this a more strongly tyepd overload.  it now takes ProjectStateChecksums instead of a random checksum instance.